Doc/whatsnew/2.0.rst

```diff
@@ -671,9 +671,9 @@ errors.  If you absolutely must use 2.0 but can't fix your code, you can edit
 ``NO_STRICT_LIST_APPEND`` to preserve the old behaviour; this isn't recommended.
 
 Some of the functions in the :mod:`socket` module are still forgiving in this
-way.  For example, :func:`socket.connect( ('hostname', 25) )` is the correct
-form, passing a tuple representing an IP address, but :func:`socket.connect(
-'hostname', 25 )` also works. :func:`socket.connect_ex` and :func:`socket.bind`
+way.  For example, ``socket.connect( ('hostname', 25) )`` is the correct
+form, passing a tuple representing an IP address, but ``socket.connect('hostname', 25)``
+also works. :meth:`~socket.socket.connect_ex` and :meth:`~socket.socket.bind`
```

Comment:
Maybe add "```Methods ```" or "```:class:`socket.socket` methods ```" at the beginning of the phrase if we remove the class name?

Reply:
The paragraph starts "ome of the functions in the [socket](https://cpython-previews--110114.org.readthedocs.build/en/110114/library/socket.html#module-socket) module are still forgiving in this way.", so I think this is OK? Ideally I wouldn't've removed the class name, but Sphinx doesn't give the option.

Reply:
`connect_ex()` and `bind()` are not functions, they are methods. But wait, `connect()` also is a method. This was written before `socket.socket` became a class, so this is forgivable.

You can keep the class name when write ``` :meth:`socket.connect_ex <socket.socket.connect_ex>` ```. It is verbose, but it was used many times.